<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GaldurSettingsDialog</class>
 <widget class="QDialog" name="GaldurSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>653</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Labrstim Client</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QGroupBox" name="generalBox">
     <property name="title">
      <string>Module &amp;&amp; Connection</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QComboBox" name="portsComboBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="cbStartImmediately">
        <property name="text">
         <string>Start stimulation immediately</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="generalWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>8</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Stimulation Type</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QComboBox" name="stimTypeComboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <item>
            <property name="text">
             <string>SWR Stimulation</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Theta Stimulation</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Train Stimulation</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Spikerate Stimulation</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>General Settings</string>
        </property>
        <layout class="QFormLayout" name="formLayout_4">
         <property name="horizontalSpacing">
          <number>6</number>
         </property>
         <property name="verticalSpacing">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="samplingRateLabel">
           <property name="text">
            <string>Sampling Rate</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="samplingRateSpinBox">
           <property name="suffix">
            <string> sps</string>
           </property>
           <property name="minimum">
            <number>200</number>
           </property>
           <property name="maximum">
            <number>200000</number>
           </property>
           <property name="value">
            <number>20000</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="pulseDurationLabel">
           <property name="text">
            <string>Pulse Duration</string>
           </property>
           <property name="buddy">
            <cstring>pulseDurationSpinBox</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="pulseDurationSpinBox">
           <property name="suffix">
            <string> ms</string>
           </property>
           <property name="singleStep">
            <double>0.200000000000000</double>
           </property>
           <property name="value">
            <double>0.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="laserIntensityLabel">
           <property name="text">
            <string>Laser Intensity</string>
           </property>
           <property name="buddy">
            <cstring>laserIntensitySpinBox</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDoubleSpinBox" name="laserIntensitySpinBox">
           <property name="suffix">
            <string> V</string>
           </property>
           <property name="singleStep">
            <double>0.500000000000000</double>
           </property>
           <property name="value">
            <double>0.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="randomIntervalsGB">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Random Interval Stimulation</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <layout class="QFormLayout" name="formLayout_2">
         <property name="horizontalSpacing">
          <number>6</number>
         </property>
         <property name="verticalSpacing">
          <number>8</number>
         </property>
         <property name="topMargin">
          <number>8</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="randomIntervalLabel">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Randomi&amp;zed Interval</string>
           </property>
           <property name="buddy">
            <cstring>randomIntervalCheckBox</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="randomIntervalCheckBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="minimumIntervalLabel">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>&amp;Minimum Interval</string>
           </property>
           <property name="buddy">
            <cstring>minimumIntervalSpinBox</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="minimumIntervalSpinBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="suffix">
            <string> ms</string>
           </property>
           <property name="minimum">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="maximumIntervalLabel">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Maximum Inte&amp;rval</string>
           </property>
           <property name="buddy">
            <cstring>maximumIntervalSpinBox</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDoubleSpinBox" name="maximumIntervalSpinBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="suffix">
            <string> ms</string>
           </property>
           <property name="minimum">
            <double>2.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_3" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::MinimumExpanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="swrStimPage">
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="rightMargin">
         <number>6</number>
        </property>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QWidget" name="widget_4" native="true">
             <layout class="QFormLayout" name="formLayout">
              <item row="0" column="0">
               <widget class="QLabel" name="swrRefractoryTimeLabel">
                <property name="text">
                 <string>SWR Refractory Time</string>
                </property>
                <property name="buddy">
                 <cstring>swrRefractoryTimeSpinBox</cstring>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QDoubleSpinBox" name="swrRefractoryTimeSpinBox"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="swrPowerThresholdLabel">
                <property name="text">
                 <string>SWR Power Threshold</string>
                </property>
                <property name="buddy">
                 <cstring>swrPowerThresholdDoubleSpinBox</cstring>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QDoubleSpinBox" name="swrPowerThresholdDoubleSpinBox"/>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="convolutionPeakThresholdLabel">
                <property name="text">
                 <string>&amp;Convolution Peak Threshold</string>
                </property>
                <property name="buddy">
                 <cstring>convolutionPeakThresholdSpinBox</cstring>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QDoubleSpinBox" name="convolutionPeakThresholdSpinBox"/>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_3">
             <property name="title">
              <string>Stimulation Delay</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <widget class="QCheckBox" name="stimDelayCheckBox">
                <property name="text">
                 <string>Enabled</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Use the random intervals&lt;br/&gt;to set the delay between&lt;br/&gt;SWR detection and stimulation.
</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="thetaStimPage">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QWidget" name="widget_5" native="true">
          <layout class="QFormLayout" name="formLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="thetaPhaseLabel">
             <property name="text">
              <string>Theta Phase</string>
             </property>
             <property name="buddy">
              <cstring>thetaPhaseSpinBox</cstring>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QDoubleSpinBox" name="thetaPhaseSpinBox">
             <property name="maximum">
              <double>360.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="trainStimPage">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QWidget" name="widget_6" native="true">
          <layout class="QFormLayout" name="formLayout_5">
           <item row="0" column="0">
            <widget class="QLabel" name="trainFrequencyLabel">
             <property name="text">
              <string>Fre&amp;quency</string>
             </property>
             <property name="buddy">
              <cstring>trainFrequencySpinBox</cstring>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QDoubleSpinBox" name="trainFrequencySpinBox">
             <property name="minimum">
              <double>1.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="spikeStimPage">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>4</number>
        </property>
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item>
         <layout class="QFormLayout" name="formLayout_6">
          <item row="0" column="0">
           <widget class="QLabel" name="detectionTimeWindowLabel">
            <property name="text">
             <string>Detection Time Window</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="spikeTimeWindowSpinBox">
            <property name="suffix">
             <string> ms</string>
            </property>
            <property name="minimum">
             <number>10</number>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
            <property name="value">
             <number>1000</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="triggerFrequencyLabel">
            <property name="text">
             <string>Trigger Frequency</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="spikeTriggerFrequencySpinBox">
            <property name="suffix">
             <string> Hz</string>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>200</number>
            </property>
            <property name="value">
             <number>20</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="spikeStimCooldownTimeLabel">
            <property name="text">
             <string>Stimulation Cooldown Time</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="spikeCooldownTimeSpinBox">
            <property name="suffix">
             <string> ms</string>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>30000</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="spikeThresholdValueLabel">
            <property name="text">
             <string>Spike Threshold Value</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSpinBox" name="spikeThresholdSpinBox">
            <property name="minimum">
             <number>-25000</number>
            </property>
            <property name="maximum">
             <number>0</number>
            </property>
            <property name="value">
             <number>-2500</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>390</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QPlainTextEdit" name="logViewBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>randomIntervalCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>minimumIntervalSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>559</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>602</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>randomIntervalCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>maximumIntervalSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>559</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>602</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>randomIntervalCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>minimumIntervalLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>559</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>525</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>randomIntervalCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>maximumIntervalLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>559</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>525</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stimTypeComboBox</sender>
   <signal>activated(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>229</y>
    </hint>
    <hint type="destinationlabel">
     <x>675</x>
     <y>409</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stimDelayCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>randomIntervalCheckBox</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>478</x>
     <y>370</y>
    </hint>
    <hint type="destinationlabel">
     <x>559</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stimDelayCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>randomIntervalCheckBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>478</x>
     <y>370</y>
    </hint>
    <hint type="destinationlabel">
     <x>559</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
